packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -336,12 +336,12 @@ export default class SrDisks extends Component {
 
   // the warning will be displayed if the SR is local
   // or the VDIs contain at least one VBD.
-  _getWarningBeforeMigrate = createSelector(
+  _getGenerateWarningBeforeMigrate = createSelector(
     createCollectionWrapper(_ => _),
     vdis => sr =>
       sr === undefined ||
       isSrShared(sr) ||
-      every(vdis, _ => isEmpty(_.$VDBs)) ? null : (
+      every(vdis, _ => _.$VBDs.length === 0) ? null : (
```

Comment:
We need to understand why it doesn't work with `isEmpty`.

Reply:
It's my fault and it works.